app/src/components/main/grid/gridBody/nestedGridRow/nestedGridRow.jsx

```diff
@@ -95,6 +95,8 @@ LoadButton.defaultProps = {
   customClassName: '',
 };
 
+// The circular dependency will not be fixed because there is a direct dependency of this component on NestedGridBody.
```

Comment:
Please also add a `CircularDependencyPlugin` exception for these files to avoid considering them as false positive warnings.
Can be done in the `app/webpack/dev.config.js` via `exclude` Regex.

Reply:
I added the name of the gridBody folder to the regex. Full path does not work:
/a\.js|node_modules|src\/components\/main\/grid\/gridBody\/nestedGridBody\.jsx/
/a\.js|node_modules|src\/components/ - It doesn't even work like that